doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -30,6 +30,17 @@ The following changes are mandatory to make your application work in the same wa
       * The ``CONFIG_CHIP_SED_ACTIVE_INTERVAL`` Kconfig option was renamed to :kconfig:option:`CONFIG_CHIP_ICD_FAST_POLLING_INTERVAL`.
       * The ``CONFIG_CHIP_SED_ACTIVE_THRESHOLD`` Kconfig option was renamed to :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_THRESHOLD`.
 
+* For applications using the Modem SLM library:
+
+  * GPIO has been changed from ``gpio0`` to configurable via devicetree. Add following to devicetree for ``gpio0`` usage:
```

Comment:
```suggestion
  * The GPIO used is now configured using devicetree instead of always using ``gpio0``.
    Add the following Zephyr ``chosen`` to your application devicetree to use ``gpio0``:
```